### DISCUSSION TITLE
OnlyOffice in AIO

### DISCUSSION THREAD

**Discussion by DasCanard** [nextcloud/all-in-one#3560] [Questions] [answered]
I'm currently not entirely satisfied with Collabora (Nextcloud Office). However, I have had many good experiences with Onlyoffice in the past.

I did a little research and saw that onlyoffice exists in the AIO but is no longer available for selection.

Is there a way to get to onlyoffice via the AIO? Since it is still being updated in the repo.

Best regards

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/2461
